src/adapter/graphql/tests/tests/test_accounts.rs

```diff
@@ -89,6 +89,36 @@ async fn test_account_by_name() {
             }
         })
     );
+
+    let res = schema
+        .execute(
+            async_graphql::Request::new(format!(
+                r#"
+                query {{
+                    accounts {{
+                        byName (name: "{}") {{
+                            accountName
+                        }}
+                    }}
+                }}
+                "#,
+                DEFAULT_ACCOUNT_NAME.to_ascii_uppercase(),
```

Comment:
I think we need a test that:
- creates a dataset under `KaMu/DaTa` account and name
- fetches dataset from GQL as `kAmU/dAtA`
- and then checks that `alias` in response is still `KaMu/DaTa`

Reply:
Added tests